ac1/_sources/sql/how_to_run_sql.rst

```diff
@@ -0,0 +1,36 @@
+.. Copyright (C)  Google, Runestone Interactive LLC
+   This work is licensed under the Creative Commons Attribution-ShareAlike 4.0
+   International License. To view a copy of this license, visit
+   http://creativecommons.org/licenses/by-sa/4.0/.
+
+
+How to Run SQL Outside of This Textbook
```

Comment:
"Outside Runestone" would be better

Reply:
I tried to keep the language more general here so that it would apply to a jupyter notebook/markdown version of the textbook as well. I changed it to runestone for now, but what do you think for this case?

Reply:
I agree with you actually. I will change it back directly and merge the PR. Thanks!